package Exception_Handling;

import java.util.Scanner;

public class ExceptionDemo1 {
    public static void main(String[] args) {
        int a,b;
        Scanner S = new Scanner(System.in);
        a = S.nextInt();
        b = S.nextInt();
//        try{
//            System.out.println("Division Result:" + (a/b));
//        }finally{
//            System.out.println("Program Completes Successfully......");
//        }
        //FileReader f = new FileReader("java.txt");
        
    }
}
